source/features/releases-tab.tsx

```diff
@@ -15,19 +16,22 @@ import {buildRepoURL, getRepo} from '../github-helpers';
 
 const getCacheKey = (): string => `releases-count:${getRepo()!.nameWithOwner}`;
 
-function parseCountFromDom(): number {
+async function parseCountFromDom(): Promise<number> {
 	const releasesCountElement = select('.numbers-summary a[href$="/releases"] .num');
 	if (releasesCountElement) {
 		return looseParseInt(releasesCountElement);
 	}
 
-	// In "Repository refresh" layout, look for the releases link in the sidebar
-	const moreReleasesCountElement = select('[href$="/tags"] strong');
-	if (moreReleasesCountElement) {
-		return looseParseInt(moreReleasesCountElement);
+	// In "Repository refresh" layout, look for the tags link in the header
+	const repositoryHeader = (await elementReady('.repository-content .file-navigation', {waitForChildren: false}))!;
```

Comment:
Maybe `{waitForChildren: true}` would circumvent the need for `oneMutation`? Or is this the same pitfall as before?

Reply:
I'm confused as to why/when `oneMutation` would be required at all. The value seems to be in the Dom on load, so the only fix in this PR should be to use `elementReady` instead of `select-dom`.

Have you found a situation where the content is loaded later via `include-fragment`? If so, let's target that `include-fragment` directly, like we do in other features.

Reply:
> Have you found a situation where the content is loaded later via include-fragment?

No. I guess I misconstrued your previous comment? ↓

> Keep in mind that by default elementReady does not work for lazy-loaded elements and the option to do so is unsafe. You'll have to rely on whatever load event they fire or use an observer: simplified, oneMutation or selector-observer

I may be wrong and it's not really lazy-loaded. After all `elementReady` seemed to solve the problem when I tested it.